package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "evm-lite-env",
+  "version": "0.1.0",
+  "description": "Deploy environment for running Lightweight Ethereum node with interchangeable consensus.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/DLTcollab/evm-lite-env.git"
+  },
+  "author": "BiiLabs",
```

Comment:
The author would be "BiiLabs, Co. Ltd.`".

Reply:
I have corrected the naming more rigorous.